pandas/io/pytables.py

```diff
@@ -332,10 +336,6 @@ def read_hdf(
         Return an iterator object.
     chunksize : int, optional
         Number of rows to include in an iteration when using an iterator.
-    errors : str, default 'strict'
-        Specifies how encoding and decoding errors are to be handled.
-        See the errors argument for :func:`open` for a full list
-        of options.
```

Comment:
The CI complained that this is in a different order than in the signature, so I moved it to be after ``mode`` like in the signature, which must be correct.

I got no idea why this is failing now and just in this PR...